source/ui/Components/Output/SevenSegment/SevenSegmentSegment.qml

```diff
@@ -22,11 +22,11 @@ import QtQuick 2.0
 import QtQuick.Dialogs 1.0
 
 /*
-Represent one segment for a 7-segment display:
+Represents one generic segment for a 7-segment display:
```

Comment:
`SegmentSegment` is maybe not the best chosen name (`SevenSegmentEdge/Line/Component` or something?)